src/index.html

```diff
@@ -6,9 +6,17 @@
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Snake</title>
-  <link rel="stylesheet" href="./style.css">
+  <link rel="stylesheet" href="style.css">
 </head>
 <body>
-  <h1>Snake</h1>
+  <div class="container">  
+      <div class="square square--1">1</div>
+      <div class="square square--2">2</div>
+      <div class="square square--3">3</div>
+      <div class="square square--4">4</div>
+      <div class="square square--5">5</div>
+      <div class="square square--6">6</div>
```

Comment:
```suggestion
      <div class="square">1</div>
      <div class="square">2</div>
      <div class="square">3</div>
      <div class="square">4</div>
      <div class="square">5</div>
      <div class="square">6</div>
```
Используй nth-child вместо отдельных классов

Reply:
В завданні написано
екстра класес.
Я ж їх не з фонаря туди написав.
Ви по різному перевіряєте і кожен говорить зробити по своєму. 
Я нічого не роблю лиш одну і ту саму задачу переробляю.

Reply:
В завданні не сказано нічого про nth-child